examples/Firebase_ESP8266_Neopixel/web/web-demo.html

```diff
@@ -0,0 +1,226 @@
+<!--
+
+  Copyright 2016 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<html>
+<head>
+  <script src="https://cdn.firebase.com/js/client/2.2.1/firebase.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.0/jquery.min.js"></script>
+  <script src="bower_components/webcomponentsjs/webcomponents-lite.js"></script>
+  <link rel="import" href="./bower_components/polymer-color-picker/polymer-color-picker.html">
+</head>
+<script>
+  // TODO: Needs security.
+  var firebaseUrl = 'https://brilliant-fire-9638.firebaseio.com';
+  var firebaseSecret = 'llXzJtqwdiHCHw6F6sehQR2EPwbO0YYdhEjPRZxG';
+</script>
+<body>
+<h2>Paint Demo</h3>
+  <polymer-color-picker onClick="updateColor('colorpicker')" id="colorpicker"
+      color="{{selectedColor}}"></polymer-color-picker>
+   Pick a color then click a cell.
+
+  <!-- TODO: Switch to CSS, table is so basic. -->
+  <table border=1 style="width:800px; height:400px">
+    <tr>
+      <td id="pixel0" onClick="setColor('pixel0')"></td>
+      <td id="pixel1" onClick="setColor('pixel1')"></td>
```

Comment:
maybe you could use [paper-button](https://elements.polymer-project.org/elements/paper-button?view=demo:demo/index.html&active=paper-button)?